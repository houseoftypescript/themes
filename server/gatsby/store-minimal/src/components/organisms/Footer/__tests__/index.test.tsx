import { render } from '@testing-library/react';
import { Footer } from '../index';
import React from 'react';

describe('Footer', () => {
  it('to match snapshot', () => {
    const { container } = render(<Footer title="" />);
    expect(container).toMatchSnapshot();
  });
});
