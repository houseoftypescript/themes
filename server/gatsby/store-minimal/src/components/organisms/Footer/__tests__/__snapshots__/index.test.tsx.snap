// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer to match snapshot 1`] = `
<div>
  <footer>
    <div
      class="py-8 border-t"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="flex justify-between"
        >
          <div>
            Â© 
            2023
             
          </div>
          <div
            class="flex gap-4"
          >
            <a
              href="#"
              target="_blank"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="FacebookIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M5 3h14a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2m13 2h-2.5A3.5 3.5 0 0 0 12 8.5V11h-2v3h2v7h3v-7h3v-3h-3V9a1 1 0 0 1 1-1h2V5z"
                />
              </svg>
            </a>
            <a
              href="#"
              target="_blank"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="InstagramIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7.8 2h8.4C19.4 2 22 4.6 22 7.8v8.4a5.8 5.8 0 0 1-5.8 5.8H7.8C4.6 22 2 19.4 2 16.2V7.8A5.8 5.8 0 0 1 7.8 2m-.2 2A3.6 3.6 0 0 0 4 7.6v8.8C4 18.39 5.61 20 7.6 20h8.8a3.6 3.6 0 0 0 3.6-3.6V7.6C20 5.61 18.39 4 16.4 4H7.6m9.65 1.5a1.25 1.25 0 0 1 1.25 1.25A1.25 1.25 0 0 1 17.25 8 1.25 1.25 0 0 1 16 6.75a1.25 1.25 0 0 1 1.25-1.25M12 7a5 5 0 0 1 5 5 5 5 0 0 1-5 5 5 5 0 0 1-5-5 5 5 0 0 1 5-5m0 2a3 3 0 0 0-3 3 3 3 0 0 0 3 3 3 3 0 0 0 3-3 3 3 0 0 0-3-3z"
                />
              </svg>
            </a>
          </div>
        </div>
      </div>
    </div>
  </footer>
</div>
`;
